k8s/jenkins/Makefile

```diff
@@ -58,29 +55,19 @@ app/build:: .build/jenkins/deployer \
 	    --tag "$(APP_DEPLOYER_IMAGE)" \
 	    -f deployer/Dockerfile \
 	    .
-
 	docker push "$(APP_DEPLOYER_IMAGE)"
 	@touch "$@"
 
 
 .build/jenkins/jenkins: .build/var/REGISTRY \
                         .build/var/TAG \
                         | .build/jenkins
-
-	docker pull "marketplace.gcr.io/google/jenkins2:$(TAG)"
-	docker tag "marketplace.gcr.io/google/jenkins2:$(TAG)" \
+	docker pull marketplace.gcr.io/google/jenkins2:$(TAG)
+	docker tag marketplace.gcr.io/google/jenkins2:$(TAG) \
 	    "$(REGISTRY)/jenkins:$(TAG)"
 	docker push "$(REGISTRY)/jenkins:$(TAG)"
 	@touch "$@"
 
-.build/jenkins/prometheus-to-sd: .build/var/REGISTRY \
```

Comment:
Please add an issue to track addition of metrics exporter for Jenkins.

Reply:
I think we have recently removed metrics from Jenkins. Do you want to add them again?